pom.xml

```diff
@@ -33,7 +33,7 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
         <graylog2.plugin-dir>/usr/share/graylog-server/plugin</graylog2.plugin-dir>
-        <graylog2.version>2.0.0</graylog2.version>
+        <graylog2.version>2.3.0-beta.1</graylog2.version>
```

Comment:
I'll revert it. I was using "stream.getAlertConditions()", which was deprecated and removed, hence the API upgrade.